### DISCUSSION TITLE
Error Retrieving Translations

### DISCUSSION THREAD

**Discussion by rwlove** [zachowj/node-red-contrib-home-assistant-websocket#513] [Q&A]
I cannot seem to get node-red to sync with Home Assistant.

I installed the node-red custom component in Home Assistant.

```
ls /mnt/kubernetes/home-assistant/config/custom_components/nodered/ -l
total 60
-rw-r--r-- 1 root root 2072 Oct 14 11:24 binary_sensor.py
-rw-r--r-- 1 root root  983 Oct 14 11:24 config_flow.py
-rw-r--r-- 1 root root 1445 Oct 14 11:24 const.py
-rw-r--r-- 1 root root 3382 Oct 14 11:24 discovery.py
-rw-r--r-- 1 root root 7501 Oct 14 11:24 __init__.py
-rw-r--r-- 1 root root  391 Oct 14 11:24 manifest.json
drwxr-xr-x 2 root root  163 Oct 14 11:26 __pycache__
-rw-r--r-- 1 root root 1172 Oct 14 11:24 sensor.py
-rw-r--r-- 1 root root  747 Oct 14 11:24 services.yaml
-rw-r--r-- 1 root root 8209 Oct 14 11:24 switch.py
drwxr-xr-x 2 root root   36 Oct 14 11:24 translations
-rw-r--r-- 1 root root  546 Oct 14 11:24 utils.py
-rw-r--r-- 1 root root 5882 Oct 14 11:24 websocket.py
```

I installed node-red-contrib-home-assistant-websocket via node-red's Pallate.

I created a long lived access token in Home Assistant.

In node-red, I dragged a 'device' onto the workspace and edited the Server.

I added the URL and token and clicked 'Update' and I get a popup dialog that says, "Error Retrieving Translations."

I don't see anything in the node-red or Home Assistant (debug turned on) logs.

What can I do to debug this issue?

I am running these programs in a k8s@home cluster, so there are some complicating factors, but many others have this working.

Any suggestions would be greatly appreciated!

**Comment by zachowj** [comment]
If you put a `events: all` node in the workspace and deploy does it show `running` underneath it?
Are you sure the custom component is installed? Did you complete step 8 <https://github.com/zachowj/hass-node-red#installation>

**Reply by rwlove** [reply]
Yes, I did install Node-Red Companion, sorry for omitting it from my original post.

...and it's working.

I think the problem was that I never actually hit 'deploy' because I thought I was blocked by 'Error Receiving Translations." Once I added ```events: all``` and deployed I was able to add the 'device trigger' object and the Device drop-down-box is now populated.

Thanks, I've been struggling with this for a while, it's funny that it was such a simple fix.

**Reply by Xeevis** [reply]
> Thanks, I've been struggling with this for a while, it's funny that it was such a simple fix.

Exactly! Just came across the same issue, the error seemed like something isn't working and it wasn't obvious you need to deploy before things start to populate.
